jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy

```diff
@@ -0,0 +1,48 @@
+job('trigger_host_os_iso_build_from_builds_repo') {
+  label('!master')
+  logRotator {
+    numToKeep(30)
+  }
+  concurrentBuild()
+  throttleConcurrentBuilds {
+    maxPerNode(1)
+  }
+  parameters {
+    stringParam('sha1', '', 'SHA-1 of the commit to build.')
+  }
+  properties {
+    githubProjectUrl("https://github.com/${GITHUB_ORGANIZATION_NAME}/builds/")
+  }
+  triggers {
+    githubPullRequest {
+      userWhitelist("${GHPRB_ADMIN_USER}")
+      orgWhitelist("${GHPRB_ADMIN_ORGANIZATION}")
+      allowMembersOfWhitelistedOrgsAsAdmin()
+      cron('H/5 * * * *')
+      triggerPhrase('.*start\\W+(tests|build_iso).*')
```

Comment:
I used `triggerPhrase('.*start\\W+iso.*')` in the 'trigger_host_os_iso_from_versions' trigger. Also, the triggers for 'build_host_os' all use ` triggerPhrase('.*start\\W+(tests|build).*')`. That '.*' there in the end would make this job trigger a second 'build_host_os'